WordPress/src/main/java/org/wordpress/android/ui/blaze/blazepromote/BlazePromoteParentActivity.kt

```diff
@@ -40,6 +63,8 @@ class BlazePromoteParentActivity : LocaleAwareActivity() {
                     supportFragmentManager.beginTransaction()
                         .replace(R.id.container, BlazePromoteWebViewFragment.newInstance())
                         .commitNow()
+                    // without this, rotating the screen will show the overlay again
+                    shouldShowOverlay = false
```

Comment:
Noting that this appears to also dismiss the overlay if the device is rotated while it is displayed. Likely not a significant issue, but could be disruptive is someone accidentally rotates their device while reading the overlay. Unsure if this is worth spending time to address.